REVHubInterface/__main__.py

```diff
@@ -789,6 +519,7 @@ def __init__(self, root):
         style = tkinter.ttk.Style()
         style.configure("Red.Label",   foreground="red")
         style.configure("Green.Label", foreground="green")
+        style.configure("Quit.TButton", foreground='red', font=('TkDefaultFont', 16, 'bold'))
```

Comment:
I think the bold text and font size increase is too much here, just making it red would be fine. I found it a little bit distracting and out of place with the large text and bold.